src/components/DatePickerWithInput/DatePickerWithInput.tsx

```diff
@@ -13,10 +13,14 @@ export interface DatePickerWithInputProps
 }
 
 export const DatePickerWithInput = (props: DatePickerWithInputProps) => {
-  const { value, onChange, ...rest } = props;
-
+  const { value, onChange, className, ...rest } = props;
   const [open, setOpen] = React.useState(false);
 
+  let inputClassName = 'grafana-plugin-ui-date-input'
+  if (className) {
+    inputClassName += ` ${className}`;
+  }
+
   return (
     <>
```

Comment:
Just for safety, don't want to change the markup as it might be a breaking change.

Reply:
With the change I made I just kind of scoped the styles so that they apply only to the specific inputs. And the styles became even more specific than before. So don't see a need for `!important` at the moment.